microscopium/metrics.py

```diff
@@ -154,3 +154,45 @@ def gene_distance_score(X, collection, metric='euclidean'):
     all_intragene_data = distance[all_intragene_index]
     all_intergene_data = distance[all_intergene_index]
     return all_intragene_data, all_intergene_data
+
+
+def partition_range(z1, z2, n):
+    #builds a partition of bins over the entire range of z1 and z2
+    #Params: z1 and z2 are arrays to be concatenated
+    #n = number of bins
+    #Returns an array of bin edges of length n+1
+
+    eps = 1e-30
+    d_max = max(np.max(z1), np.max(z2)) + eps
+    d_min = min(np.min(z1), np.min(z2))
+    partition = np.linspace(d_min, d_max, n) #or n, check this
+
+    return partition
+
+
+def empirical_distribution(values,bins):
```

Comment:
Prepended _ to empirical_distribution and partition_range
Added space after comma